pkg/sql/backfill/backfill.go

```diff
@@ -640,11 +641,12 @@ func (ib *IndexBackfiller) InitForDistributedUse(
 	ctx context.Context,
 	flowCtx *execinfra.FlowCtx,
 	desc catalog.TableDescriptor,
+	indexesToBackfill []catid.IndexID,
 	mon *mon.BytesMonitor,
 ) error {
 
 	// Initialize ib.added.
-	ib.initIndexes(desc)
+	ib.initIndexes(desc, indexesToBackfill)
```

Comment:
Is there a way to test this? I think probably a unit test for the `initIndexes` method is sufficient, that logic here should trivial, but nice to make sure the method actually set thing rithg.